package.json

```diff
@@ -22,7 +22,7 @@
   "license": "MPL-2.0",
   "repository": "mozilla/notes",
   "scripts": {
```

Comment:
@udaraweerasinghege Let's add a script command as well. Maybe `npm run start-rtl` that would use the `ar`locale for now